docs/data/data.rst

```diff
@@ -1,39 +1,237 @@
+  The resulting surface reflectance values are comparable both within individual images and between images acquired at different times and/or with different sensors.
+
+* Terrain affects optical satellite images through both irradiance and bidirectional reflectance distribution function (BRDF) effects.
+* Slopes facing the sun receive enhanced solar irradiance and appear brighter compared to those facing away from the sun.
+* For anisotropic surfaces, the radiance received at the satellite sensor from a sloping surface is also affected by surface
+  BRDF which varies with combinations of surface landcover types, sun, and satellite geometry (sun and sensor view, and their relative
+  azimuth angle) as well as topographic geometry (primarily slope and aspect angles).
+  Consequently, to obtain comparable surface reflectance from satellite images covering mountainous areas,
+  it is necessary to process the images to reduce or remove the topographic effect so that the images can be used for different purposes on the same spectral base.
+* A Digital Surface Model (DSM) resolution appropriate to the scale of the resolution of satellite image is needed for the best results. 1 second SRTM DSM is
+  used for NBART processing.
+
+It has the same bands description provided in Surface reflactance NBAR table_.
+
+Landsat Archive
 ---------------
-NBAR stands for **Nadir-corrected BRDF Adjusted Reflectance**.  It is the surface reflectance product in
+GA has acquired Landsat imagery over Australia since 1979, including  TM, ETM+ and OLI imagery. While this data has been used extensively for numerous
+land and coastal mapping studies, its utility for accurate monitoring of environmental resources has been limited by the processing methods that have been traditionally
+used to correct for inherent geometric and radiometric distortions in EO imagery.
+To improve access to Australia s archive of Landsat TM/ETM+/OLI data, several collaborative projects have been undertaken in conjunction with industry, government and academic partners.
+These projects have enabled implementation of a more integrated approach to image data correction that incorporates
+normalising models to account for atmospheric effects, BRDF and topographic shading (Li et al., 2012).
+
+The approach has been applied to Landsat TM/ETM+ and OLI imagery to create the SR products. The advanced supercomputing facilities provided by the National
+Computational Infrastructure (NCI) at the Australian National University (ANU) have been instrumental in handling the considerable data volumes and processing
+complexities involved with production of this product.
 
-It has the bands:
+Surface Reflectance Correction Models
+--------------------------------------
+    Image radiance values recorded by passive EO sensors are a composite of:
 
-===== ========================
```

Comment:
While Landsat has `panchromatic`, `cirrus` and `tirs` bands, they are at different resolutions and stages of processing, so we don't include them in out tiled `ls*_nbar_albers` products.